{"name": "API 키가 잘못되었을 때 401 Unauthorized 확인", "status": "passed", "start": 1744514745568, "stop": 1744514745633, "uuid": "cee83d32-622b-4452-a4a9-8dffe5e06a6f", "historyId": "a386eac7b0b41582b56dc8e54e039987", "testCaseId": "a386eac7b0b41582b56dc8e54e039987", "fullName": "src.tests.api_tests.test_moive_invalid_api_key#test_movie_invalid_api_key", "labels": [{"name": "feature", "value": "예외 케이스"}, {"name": "story", "value": "잘못된 API 키"}, {"name": "parentSuite", "value": "src.tests.api_tests"}, {"name": "suite", "value": "test_moive_invalid_api_key"}, {"name": "host", "value": "fv-az790-834"}, {"name": "thread", "value": "3563-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.tests.api_tests.test_moive_invalid_api_key"}]}